dockerfiles/Dockerfile-ui

```diff
@@ -1,4 +1,4 @@
-FROM node:14.17.3-alpine3.12 as FRONTEND
+FROM node:14.17.3-alpine3.15 as FRONTEND
```

Comment:
The latest node alpine version is 3.14. Could also update the node version to 14.17.6.